code/src/index.css

```diff
@@ -1,13 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  margin: 40px 20px;
+  font-family: 'Roboto Mono', monospace; 
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.form {
+  display: flex;
+  flex-direction: column;
+  border: solid 1px;
+  box-shadow: 5px 7px;
+  background: #e3e2e7;
+}
+
+.form-title {
+  font-size: 16px;
+  font-weight: bold;
+  padding-left: 10px;
+  margin-top: 10px;
+}
+
+.input {
+  margin: 10px;
+  padding: 20px;
+  height: 30px;
+  width: 80%;
+  vertical-align: top;
+  overflow-y: auto;
+  overflow-wrap: break-word;
+  resize: none;
+}
+
+.thought-button {
+  width: 65%;
+  padding: 10px;
+  margin: 5px 0 20px 10px;
+  border-radius: 20px;
+  background-color:#fcbacb;
+  font-weight: bold;
+  border: none;
+  outline: none;
+}
+
+.thought-button:hover {
+  background: #f97194;
+}
+
+.article {
+  display: flex;
+  flex-wrap: wrap;
+  border: solid 1px;
+  box-shadow: 5px 7px;
+  margin-top: 40px; 
+}
+
+.thought-message {
+  font-size: 18px;
+  font-weight: 600;
+  width: 80%;
+  padding-left: 10px;
+  margin-top: 10px;
+  margin-bottom: 20px;
+  word-wrap: break-word; /* Allow long words to be able to break and wrap onto the next line */
```

Comment:
I am a big fun of comments, because it helps me when I am writing a code to understand better what I am doing. And also when I return to my code later on comments are super helpful. I can see that you have the same approach, it is great ✔✨